101-machine-learning-linkedserivce-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "amlWorkspaceName": {
+            "value": "GET-PREREQ-amlWorkspaceName"
+        },
+        "linkName": {

```

Comment:
remove this and use a defaultValue in the template

Reply:
done.